### PR TITLE
Load more courses Done

### PR DESCRIPTION
**Pull request by Dhroov7**
### Is this PR regarding CSS? 
This project uses motley(https://github.com/coding-blocks/motley/) as css framework. We don't accept css changes here. Please consider a PR to motley instead

### CODE REVIEW

**Comment by Dhroov7** [comment]
@witty123 Please review.

**Comment by witty123** [comment]
Dashed button at centre and a little bigger, refer doubt section in motley

File: app/pods/courses/index/template.hbs [hunk 1]
```diff
@@ -88,11 +88,12 @@
     {{/each}}
   </div>
   <div class="row">
-    <div class="col-10"></div>
+    <div class="col-5"></div>
```
Comment (witty123) [inline app/pods/courses/index/template.hbs]: Use offset please

**Comment by Dhroov7** [comment]
@witty123 Done with the changes.

File: app/pods/courses/index/template.hbs [hunk 2]
```diff
@@ -87,4 +87,15 @@
       </div>
     {{/each}}
   </div>
+  {{#if showLoadMore}}
+    <div class="row">
+      <div class="col-5"></div>
```
Comment (abhishek97) [inline app/pods/courses/index/template.hbs]: use `offset-` or `justify-content` instead

**Comment by abhishek97** [comment]
@Dhroov7 Fix conflicts and do changes.

**Comment by witty123** [comment]
@Dhroov7 resolve conflicts please

**Pull request merged**